lib/ts/BufferWriterFormat.cc

```diff
@@ -336,36 +366,19 @@ namespace bw_fmt
     width -= static_cast<int>(n);
     string_view digits{buff + sizeof(buff) - n, n};
 
-    // The idea here is the various pieces have all been assembled, the only difference
-    // is the order in which they are written to the output.
-    switch (spec._align) {
-    case BWFSpec::Align::LEFT:
+    if(spec._align == BWFSpec::Align::SIGN) { // custom for signed 
```

Comment:
I think it might be better to add a `char sign` argument to `Write_Aligned` rather than special casing this here. Allow it to be 0 to indicate no sign, otherwise the sign character, and split out the `SIGN` alignment case.